engine/combat/utility/sos_appraisal.py

```diff
@@ -170,19 +170,19 @@ def execute_selecting_enemy_sequence(self: Self) -> None:
 
         match self.battle_command:
             case SoSBattleCommand.Attack:
-                selected_target = self.combat_manager.selected_attack_target_guid
+                pass
```

Comment:
What do these changes to pass do? Is the match statement needed at all anymore? It seems to do nothing.

Reply:
i'm coming back for a refactor on this later. In the midst of 900 refactors to get this thing going :(